expectise/expect.py

```diff
@@ -233,10 +238,24 @@ def set_up(cls, klass: Type, method_name: str, surrogate: Callable) -> None:
             "decorators": [],
             "with_args": [],
             "return": [],
+            "mock_ref": mock_ref,
         }
         # Substituting original methods with surrogates that raise errors if not preceded by `Expect` statements
         setattr(klass, method_name, surrogate)
 
+    @classmethod
+    def disable_mock(cls, class_name: str, method_name: str) -> None:
+        """Disables a Mock."""
+        if (class_name, method_name) in cls.method_h:
+            cls.method_h[(class_name, method_name)]["mock_ref"].disable()
+        else:
+            raise ValueError(
+                """
+                Method `{method_name}` from class `{class_name}` is not mocked, or does not exist.
+                Calling disable_mock has no effect.
+            """
+            )
```

Comment:
syntax suggestion: early fail and less indentation
```python
if not (class_name, method_name) in cls.method_h:
    raise(...)

cls.method_h[(class_name, method_name)]["mock"].disable()

Reply:
Much better!